src/course.py

```diff
@@ -56,6 +56,8 @@ def get_course_files(self, course_path) -> None:
         files_body = self.course_soup.find_all(class_="card-body")
 
         for item in files_body:
+            if item.find('strong') == None:
```

Comment:
```suggestion
            if not item.find('strong'):
```

Reply:
or either use `is`

Reply:
and maybe add a comment why such an if condition exists

Reply:
Thanks for the suggestion. Refactoring and documentation done.